pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -143,33 +142,37 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 		}
 		// the status map doesn't exist yet, watching below is still an OK thing to do
 	}
+	originalNodes := 0
+	remainingNodes := util.NewSet()
+	checkStatus := func() {
+		remainingNodes = util.NewSet()
+		// check the nodes to see which ones are already completed
+		originalNodes = len(statuses.Items)
+		for _, configMap := range statuses.Items {
+			node, ok := configMap.Labels[nodeLabelKey]
+			if !ok {
+				logger.Infof("missing node label on configmap %s", configMap.Name)
```

Comment:
think this should be a warn/error level log statement instead?  what would it really mean to not have a node label on one of the status config maps?